src/main/scala/beam/replanning/utilitybased/ChainBasedTourVehicleAllocator.scala

```diff
@@ -211,8 +211,7 @@ object ChainBasedTourVehicleAllocator {
           case act: Activity =>
             Option(act.getEndTime).getOrElse(
               throw new RuntimeException(
-                s"could not get " +
-                s"time from $act"
+                s"could not get time from $act"
```

Comment:
if scalaFmt allows keep in the same line
```Scala
throw new RuntimeException(s"could not get time from $act")
```

Reply:
apply this change